packages/format-library/src/text-color/index.native.js

```diff
@@ -33,9 +33,7 @@ const name = 'core/text-color';
 const title = __( 'Text color' );
 
 function getComputedStyleProperty( element, property ) {
-	const {
-		props: { style = {} },
-	} = element;
+	const style = element?.props?.style ?? {};
```

Comment:
This logic relies on the fact that the ref object (`contentRef`), passed from the `RichText` component, contains the props passed to the `AztecView` component.

https://github.com/WordPress/gutenberg/blob/a2b47844e1effb5e946597437d9e0dc00166ffbd/packages/format-library/src/text-color/index.native.js#L51-L58

https://github.com/WordPress/gutenberg/blob/a2b47844e1effb5e946597437d9e0dc00166ffbd/packages/format-library/src/text-color/index.native.js#L65-L86

https://github.com/WordPress/gutenberg/blob/a2b47844e1effb5e946597437d9e0dc00166ffbd/packages/block-editor/src/components/rich-text/native/format-edit.js#L40

https://github.com/WordPress/gutenberg/blob/a2b47844e1effb5e946597437d9e0dc00166ffbd/packages/block-editor/src/components/rich-text/native/index.native.js#L1337

https://github.com/WordPress/gutenberg/blob/a2b47844e1effb5e946597437d9e0dc00166ffbd/packages/block-editor/src/components/rich-text/native/index.native.js#L1269-L1272

In theory, the props are not exposed in ref objects, so I investigated further where are they coming from. I couldn't pinpoint the code from React/React Native but I confirmed that ref objects coming from native components (like `Aztec` or `Video (react-native-video)` contain extra properties like `props` or `forceUpdate`. On the other hand, I couldn't find any documentation related to this, so I assume this approach is not recommended and might lead to issues in future React Native versions.